### PR TITLE
coveralls.io support

### PR DESCRIPTION
**Pull request by slav0nic**
Added test coverage to  github-actions

### CODE REVIEW

**Comment by slav0nic** [comment]
what's the point of using `virtualenv` in github-actions?

**Comment by leplatrem** [comment]
> what's the point of using virtualenv in github-actions?

Not sure. None? Maybe we could get rid of it (another PR tho)

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -81,3 +81,27 @@ jobs:
         run: |
           source .venv/bin/activate
           tox -e ${{ matrix.toxenv }}
+      
+      - name: Coveralls Parallel
```
Comment (leplatrem) [inline .github/workflows/test.yml:85]: If that's possible, Instead of `Parallel`, I would use something like `Coveralls for ${{ matrix.toxenv }}`

**Review by leplatrem — APPROVED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (slav0nic) [reply .github/workflows/test.yml]: oke, done

**Pull request merged**